docs/resource-schemas.md

```diff
@@ -184,6 +215,7 @@ assets:
 | `externalDocsUrl` | Absolute URL to externally-hosted documentation.                                                                                                         | Optional | String           | No          | –             | –                                                                                                                                                                                                                                                                                                                                          |
 | `demoLinks`       | Links to demo sites. See [demo links](#demos-links).                                                                                                     | Optional | Array            | No          | –             | –                                                                                                                                                                                                                                                                                                                                          |
 | `noIndex`         | If set to `true`, the global catalogs will exclude the asset.                                                                                            | Optional | Boolean          | No          | `false`       | –                                                                                                                                                                                                                                                                                                                                          |
+| `navData`         | Links to documentation pages. See [nav data](#nav-data)                                                                                                  | Optional | Array            | -           | –             |
```

Comment:
I don't think we want to add navData to the asset resource? Per our page tabs, we just want to support usage, style, code, and accessibility content.

Reply:
I thought we had an issue for this, but it might be adding a `docs` object to asset or something, that has children `usageDocsPath`, `styleDocsPath`, `codeDocsPath`, `accessibilityDocsPath`.

Reply:
Yea you're right, removed.